EU4toV2/Source/V2World/V2World.cpp

```diff
@@ -1827,6 +1827,18 @@ void V2World::output() const
 	Utils::renameFolder("output/output", "output/" + theConfiguration.getOutputName());
 	createModFile();
 
+	// Record converter version
+
+	LOG(LogLevel::Debug) << "Writing version";
+	FILE* versionFile;
+	if (fopen_s(&versionFile, ("output/" + theConfiguration.getOutputName() + "/eu4tov2_version.txt").c_str(), "w") != 0)
+	{
+		LOG(LogLevel::Error) << "Could not create version file";

```

Comment:
Nit: I've stylistically moved towards trowing an exception and letting the overall try/catch block handle the logging and exit.

Reply:
included a try/catch block for error logging.